zerver/lib/email_mirror.py

```diff
@@ -412,20 +411,12 @@ def key_fragment(self) -> str:
     def rules(self) -> List[Tuple[int, int]]:
         return settings.RATE_LIMITING_MIRROR_REALM_RULES
 
+    def __str__(self) -> str:
+        return self.realm.name or '<Missing name>'
```

Comment:
Is `<Missing name>` a valid possibility?  I wouldn't have thought it would be...

Reply:
As a sidenote, we generally avoid `a or b` logic, preferring instead explicit if statements.  Part of the motivation is that it means test coverage helps ensure the code is actually tested, but it's also more readable.  

As another sidenote, I notice `realm.name` is apparently allowed to be null at the database level, but I'm not sure that's actually a valid possibility, and we should potentially change the database model to lock that down.

Reply:
I don't know, I found it strange, but mypy complained and I saw realm.name actually declared as Optional[str] so I did it this way. I changed this to realm.string_id now instead, to avoid any doubts.